code/modules/shuttle/emergency.dm

```diff
@@ -204,9 +204,9 @@
 		var/security_num = seclevel2num(get_security_level())
 		switch(security_num)
 			if(SEC_LEVEL_GREEN)
-				set_coefficient = 2
```

Comment:
I think make green like 12 maybe, just for some variation

Reply:
I would've said that was odd, but blue shuttle was already 12 minutes before and the amber shuttle is 8 minutes so lets just go with it.